lib/commands.rb

```diff
@@ -122,4 +122,37 @@ def execute
       end
     end
   end
+
+  class CreateDb < Etna::Command
+    usage '# create the initial database per config.yml'
+
+    def execute
+      if @no_db
+        create_db if @no_db
+
+        puts "Database is setup. Please run `bin/timur migrate #{@project_name}`."
+      else
+        puts "Database already exists."
+      end
+    end
+
+    def create_db
+      # Create the database only
+
+      puts "Creating database #{@db_config[:database]}"
+      %x{ PGPASSWORD=#{@db_config[:password]} createdb -w -U #{@db_config[:user]} #{@db_config[:database]} }
```

Comment:
Nit:
Is it possible to simply modify the `ENV['PGPASSWORD'] = @db_config[:password]` before invoking the shell command rather than placing it as part of the shell command?
Reason why is that placing this environment variable directly in the command risks leaking the password text into, say, bash history or an error log if the command invocation fails.

Reply:
Thanks, that's a good point. I think the only way is to open up a child process and run the `createdb` command there.